core/src/main/java/org/mule/VoidMuleEvent.java

```diff
@@ -40,7 +40,12 @@ public static VoidMuleEvent getInstance()
         return instance;
     }
 
-    private VoidMuleEvent()
+    public static boolean isVoid(MuleEvent event)
```

Comment:
I guess you added this method to avoid repeating the "instance of VoidMuleEvent" code, but I don't think this is the right place to put it. Also, if VoidMuleEvent can't be instantiated, this can be replaced with event == VoidMuleEvent.getInstance()